templates/image-builder.yml

```diff
@@ -183,22 +183,25 @@ objects:
     - prefix: ${FLOORIST_QUERY_PREFIX}/builds
       query: >-
         select
-          c.job_id,c.created_at,c.client_id,c.org_id,c.account_number,c.email,c.request->>'distribution' as distribution,
+          job_id,created_at,client_id,org_id,account_number,email,request->>'distribution' as distribution,
           req->>'architecture' as architecture,req->>'image_type' as image_type,req->'upload_request'->>'type' as upload_request_type,req->'ostree'->>'url' as ostree_url,
           req->>'snapshot_date' as snapshot_date,
-          v.blueprint_id::text as blueprint_id,
```

Comment:
hmm :thinking:
I'm thinking this is still pretty useful info, I'd probably keep both blueprint_id and version, so you can connect the dots.
"What blueprint and version this compose was build from" :)

It feels to me that we want to have both :)